tests/fixtures/rules/prefer-destructured-store-props/invalid/test01-errors.json

```diff
@@ -0,0 +1,38 @@
+[
+  {
+    "message": "Destructure bar from store $foo for better change tracking & fewer redraws",
+    "line": 6,
+    "column": 2,
+    "suggestions": [
+      {
+        "desc": "Using destructuring like $: ({ bar } = $foo); will run faster",
+        "messageId": "fixUseDestructuring",
+        "output": "<!-- prettier-ignore -->\n<script>\n\n$: ({ bar } = $foo);\n</script>\n\n{bar}\n{$foo[bar]}\n\n<!-- eslint-disable-next-line dot-notation -- tests -->\n{$foo.bar}\n"
+      }
+    ]
+  },
+  {
+    "message": "Destructure bar from store $foo for better change tracking & fewer redraws",
+    "line": 7,
+    "column": 2,
+    "suggestions": [
+      {
+        "desc": "Using destructuring like $: ({ bar } = $foo); will run faster",
+        "messageId": "fixUseDestructuring",
+        "output": "<!-- prettier-ignore -->\n<script>\n\n$: ({ bar } = $foo);\n</script>\n\n{$foo.bar}\n{bar}\n\n<!-- eslint-disable-next-line dot-notation -- tests -->\n{$foo.bar}\n"
```

Comment:
I think the following suggestion is wrong.

```svelte
<!-- prettier-ignore -->
<script>
$: ({ bar } = $foo);
</script>

{$foo.bar}
{bar}

<!-- eslint-disable-next-line dot-notation -- tests -->
{$foo.bar}
```

I think it should be like this:

```svelte
<!-- prettier-ignore -->
<script>
$: ({ [bar]: barValue } = $foo);
</script>

{$foo.bar}
{barValue}

<!-- eslint-disable-next-line dot-notation -- tests -->
{$foo.bar}
```

It's hard to suggest this, so I think it's better not to suggest when using `[]`.

Reply:
Fair!